tests_app/tests/functional/_examples/etags/remove_etag_gzip_postfix/tests.py

```diff
@@ -1,24 +1,26 @@
 from django.test import TestCase, override_settings
-
+from django.conf import settings
 
 @override_settings(ROOT_URLCONF='tests_app.tests.functional._examples.etags.remove_etag_gzip_postfix.urls')
 class RemoveEtagGzipPostfixTest(TestCase):
 
     @override_settings(MIDDLEWARE_CLASSES=(
```

Comment:
should this still be MIDDLEWARE_CLASSES or just MIDDLEWARE?

Reply:
I think it should be MIDDLEWARE , Because modern Django 2.0 >= use middleware instead of middleware classes .

Reply:
I will push the latest update , after running the test .

Reply:
well , I think its ok for now , I will update it to MIDDLEWARE when upgrade the version , Thanks .